### PR TITLE
feat: added 404 page

### PR DESCRIPTION
**Pull request by varenechko**
there wasn't few dependencies, i added them

### CODE REVIEW

File: src/pages/NotFound/index.tsx [hunk 1]
```diff
@@ -1 +1,29 @@
-export const NotFound = () => <p>404 - page not found</p>;
+import { Box, Button, Stack, ThemeProvider, Typography } from '@mui/material';
+import { theme } from '../../themes';
+
+
+export const NotFound = () =>
```
Comment (veatik90) [inline src/pages/NotFound/index.tsx:5]: return type FC

File: src/pages/NotFound/index.tsx [hunk 2]
```diff
@@ -1 +1,29 @@
...
+    justifyContent:'center'
+    }}
+    >
+      <Stack
+      direction="column"
+      justifyContent="center"
+      alignItems="center"
+      spacing={2}
+      >
+        <ThemeProvider theme={theme}>
```
Comment (veatik90) [inline src/pages/NotFound/index.tsx:21]: why do you need theme provider here?

File: src/pages/NotFound/index.tsx [hunk 3]
```diff
@@ -1 +1,29 @@
...
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSQAw3md2--EMzzKYG7AyImi9BeJvaJLO4riQ&usqp=CAU" alt="telescope" style={{borderRadius:'50%'}}/>
```
Comment (veatik90) [inline src/pages/NotFound/index.tsx:22]: save image in assets folder inside component and use it
get rid of inline styles. make it in styles.ts with styled component

File: src/pages/NotFound/index.tsx [hunk 4]
```diff
@@ -1 +1,29 @@
...
+            <Typography variant="subtitle1" align='center'>
+                404 | Page not found
+            </Typography>
+            <Button href='/' variant='outlined' color='primary' size='large'>home</Button>
```
Comment (veatik90) [inline src/pages/NotFound/index.tsx:26]: You refresh a page with this behavior. Check how to prevent it

File: src/pages/NotFound/index.tsx (hunk 1 above)
Comment (varenechko) [reply src/pages/NotFound/index.tsx]: fixed

File: src/pages/NotFound/index.tsx (hunk 4 above)
Comment (varenechko) [reply src/pages/NotFound/index.tsx]: fixed

**Pull request closed without merging**